org.mixedrealitytoolkit.input/Assets/Prefabs/Experimental-XRI3/SaveDevelopedPrefabsToExperimentalXRI3.ps1

```diff
@@ -0,0 +1,10 @@
+﻿Copy-Item '..\MRTK Gaze Controller.prefab' '.\MRTK XRI Gaze.prefab'
+Copy-Item '..\MRTK Interaction Manager.prefab' '.\MRTK XRI Interaction Manager.prefab'
```

Comment:
Why is this needed?  This copy is likely a onetime operation needed only be you.


Please don't include, so to avoid extra maintenance  cost.

Reply:
Reverted in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/743/commits/ae962a4080fa69b164976601408ca163bb603ee0 , resolving.